Readme.md

```diff
@@ -1,16 +1,85 @@
+<h4 align="center">MusicBrainz App</h4>
 
-# Developers
-MusicBrainz for Android was first created in 2010-11 as a part of GSoC by Jamie McDonald. The project went stagnant for a few years.
+<p align="center">
+    <a href="https://github.com/metabrainz/musicbrainz-android/commits/master">
+    <img src="https://img.shields.io/github/last-commit/metabrainz/musicbrainz-android.svg?style=flat-square&logo=github&logoColor=white"
+         alt="GitHub last commit">
+    <a href="https://github.com/metabrainz/musicbrainz-android/issues">
+    <img src="https://img.shields.io/github/issues-raw/metabrainz/musicbrainz-android.svg?style=flat-square&logo=github&logoColor=white"
+         alt="GitHub issues">
+    <a href="https://github.com/metabrainz/musicbrainz-android/pulls">
+    <img src="https://img.shields.io/github/issues-pr-raw/metabrainz/musicbrainz-android.svg?style=flat-square&logo=github&logoColor=white"
+         alt="GitHub pull requests">
+</p>
+      
+<p align="center">
+  <a href="#about">About</a> •
+  <a href="#installation">Installation</a> •
+  <a href="#downloading">Downloading</a> •
+  <a href="#developers">Developers</a> •
+  <a href="#wiki">Wiki</a> •
+  <a href="#contributing">Contributing</a> •
+  <a href="#issues">Issues</a> •
+  <a href="#support">Support</a> •
+  <a href="#license">License</a>
+</p>
+
+---
+
+## About
+
+MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Android platform. It allows you to:
+
+* View release information by scanning a barcode
+* Search for information about artists, releases, release groups,labels, recordings, instruments, and events
+* View collections
+* Tag audio files like Picard
+* Donate to the MetaBrainz Foundation via PayPal
+
+## Installation
+	    
+* First start with installing Android Studio on your pc.
+* Clone this repository.
+* Use the `gradlew build` command to build the project directly or use the IDE to run the project to your phone or the emulator.
+
+## Downloading
+
+* [Google Play Store](https://play.google.com/store/apps/details?id=org.metabrainz.android)
+	    
+* [F-Droid](https://f-droid.org/en/packages/org.metabrainz.android/)
+	 
+## Developers
+	    
+MusicBrainz App was first created in 2010-11 as a part of GSoC by Jamie McDonald. The project went stagnant for a few years.
 
 In GSoC 2019, development on the project started again and was later taken up as another GSoC project by amCap1712.
 
-In GSoC 2020,combined effort of SomalRudra & amCap1712 revamped the app design, improved overall infrastructure, integrated testing,Continous Integration & has been successful to implement the key feature:Tagger
+In GSoC 2020,combined effort of SomalRudra & amCap1712 revamped the app design, improved overall infrastructure, integrated testing,Continous Integration & has been successful to implement the key feature : Tagger
+
+## Wiki
```

Comment:
We don't have any currently. I would suggest to replace this with a link to MeB IRC channel, community and support@metabrainz.org.

Reply:
Done